js/apps/filter-page/controllers/filterPageCtrl.js

```diff
@@ -57,6 +57,7 @@
 				(filterVal || null);
 		};
 
+		//TODO: remove this in favor of the sahred helper
```

Comment:
Would you be specific here? Instead of saying "the shared helper" maybe specify `bcpl.shared.myHelper` or w/e.